ontology/omrse.md

```diff
@@ -27,6 +27,10 @@ build:
 activity_status: active
 repository: https://github.com/ufbmi/OMRSE
 preferredPrefix: OMRSE
+publications:
+  - id: https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0087-8
+    title: "CThe ontology of medically related social entities: recent developments"
+
```

Comment:
Please also add (I'm in reviewing mode, so I cannot do this myself.  Or if it's more appropriate, I can make edits later)

  - id: http://ceur-ws.org/Vol-833/paper20.pdf
    title: "Representing the reality underlying demographic data"

Reply:
@hoganwr there are currently some debates about whether to / how to add links to PDFs that appear in conferences that don't assign stable identifiers that you can get actionable metadata from in #1846 and #1855 so I would suggest holding off on that. Alternatively, you can put the link to the paper in the free text part of this markdown file.

Reply:
We could consider ceur-ws stable? Its been stable for 15 years at least.. Seems the ceur servers contain a lot of citable content

Reply:
I said the service doesn't assign stable, actionable identifiers, not that the service itself isn't stable.

Reply:
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1857